tests/test_files_directory/nested_directory/build_test_merged_file.json

```diff
@@ -1,7 +1,3 @@
 {
-    "Hello": "There",
-    "UhOh": "This",
-    "OneIs": "Nested",
-    "UhOh2": "This",
-    "OneIs2": "NestedAlso"
+    "Hello": "There"
```

Comment:
This is a badly designed unit test that takes an original file, merges it with another and saves it to the same file, overwriting it's original state. Meaning the next time unit tests run they will fail. I need to remember to clear out all these new keys every time it runs. It's a bad test we will address when improving tests